examples/Recipies/initialization/__main__.py

```diff
@@ -0,0 +1,32 @@
+"""Example to demonstrate how to implement configurable module globals and class variables
+
+Run this example with the following command:
+python ./__main__.py -c ./the_config.toml -w ./the_config.toml
+"""
+
+# the next module is placed at the top of the entry point but that does not help
+from modules.wrongly_initialized_config import WronglyInitializedConfig  # isort:skip
+
+# the next module contains configured class variables and configured module globals
+from modules.my_dataclass import MyDataclass, the_first_global, the_second_global
```

Comment:
I guess `modules` in this sense is some custom module? I liked the `my_` prefix to show that it is something external. So maybe `my_module.wrongly_init...`

Reply:
I personally think that modules is quite common and a good choice here.